### PR TITLE
Add URL support to vimeo__player.loadVideo

### PR DESCRIPTION
**Pull request by dougalg**
This PR adds support for a missing option to vimeo__player's `loadVideo`
method which can accept either a numeric video ID or a full
video URL.

See documentation here: https://developer.vimeo.com/player/sdk/reference#load-a-new-video-into-a-player

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: developer.vimeo.com/player/sdk/reference#load-a-new-video-into-a-player
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. (n/a).32

### CODE REVIEW

File: types/vimeo__player/index.d.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class Player {
 
     on(event: EventName, callback: EventCallback): void;
     off(event: EventName, callback?: EventCallback): void;
-    loadVideo(id: number): VimeoPromise<number, TypeError | PasswordError | PrivacyError | Error>;
+    loadVideo(id: number | string): VimeoPromise<number, TypeError | PasswordError | PrivacyError | Error>;
```
Comment (peterblazejewicz) [inline types/vimeo__player/index.d.ts:37]: this change is ok, the source supports and option object as well.
The only problem is, should it be versioned? (does it really work with version 2.10, the docs/readme change is newer)

**Review by gabritto — CHANGES_REQUESTED** [review]
as @peterblazejewicz pointed out, it may be the case that passing an url string to `loadVideo` is a feature introduced *after* version 2.10 of the original library. If that is the case, then you should also bump the first line in the header to something like `// Type definitions for @vimeo/player 2.16`. If that's not the case, we can merge this as is. Please let me know what you think, and then I can merge this. Thanks!

File: types/vimeo__player/index.d.ts (hunk 1 above)
Comment (dougalg) [reply types/vimeo__player/index.d.ts:37]: Good point. It's hard to say. Shall I just bump the version to match the docs?

**Comment by dougalg** [comment]
@gabritto sorry about the delay. To be on the safe side, I have done as you and @peterblazejewicz suggested and bumped the version.

Thank you!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@dougalg thx!

**Comment by dougalg** [comment]
Ready to merge

**Pull request merged**